cirq-core/cirq/ops/pauli_string.py

```diff
@@ -266,12 +266,7 @@ def __mul__(self, other: complex) -> cirq.PauliString[TKey]:
         pass
 
     def __mul__(self, other):
-        known = False
-        if isinstance(other, raw_types.Operation) and isinstance(other.gate, identity.IdentityGate):
-            known = True
-        elif isinstance(other, (PauliString, numbers.Number)):
-            known = True
-        if known:
+        if isinstance(other, (PauliString, numbers.Number)):
             return PauliString(
                 cast(PAULI_STRING_LIKE, other),
```

Comment:
Nit: I don't think this cast is needed anymore, so can be removed.

Reply:
Done